Error: Module not found: Error: Can't resolve '@angular/animations/browser

# Steps to reproduce

1. I have a plain new Angular v19 project and added Angular Testing Library using this command

```sh
ng add @testing-library/angular 
```

2. Change src/app/app.component.spec.ts content with code below

```ts
import { render, screen } from '@testing-library/angular';
import { AppComponent } from './app.component';

describe('AppComponent', () => {
  it('should render Welcome', async () => {
    await render(AppComponent);
  });
});
```

3. I then proceed to run the test using

```sh
ng test
```

# Error

```
Error: Module not found: Error: Can't resolve '@angular/animations/browser
```

**I've tried importing `BrowserAnimationsModule` but still get the same error**
```ts
import { render, screen } from '@testing-library/angular';
import { AppComponent } from './app.component';
import { BrowserAnimationsModule } from '@angular/platform-browser/animations';

describe('AppComponent', () => {
  it('should render Welcome', async () => {
    await render(AppComponent, {
      imports: [BrowserAnimationsModule],
    });
  });
});
```

## Versions
- Angular: ^19.2.0
- "@testing-library/angular": "^17.3.6",
- "@testing-library/dom": "^10.0.0",
- "@testing-library/jest-dom": "^6.4.8",
- "@testing-library/user-event": "^14.5.2",

--- a/projects/testing-library/package.json
+++ b/projects/testing-library/package.json
@@ -29,6 +29,7 @@
     "migrations": "./schematics/migrations/migrations.json"
   },
   "peerDependencies": {
+    "@angular/animations": ">= 17.0.0",
     "@angular/common": ">= 17.0.0",
     "@angular/platform-browser": ">= 17.0.0",
     "@angular/router": ">= 17.0.0",

